scripts/package.ts

```diff
@@ -106,6 +107,68 @@ function getVersionSuffix(feature: string, debug: boolean): string {
     return `${debugSuffix}${featureSuffix}${commitSuffix}`
 }
 
+/**
+ * @returns true if curl is available
+ */
+function isCurlAvailable(): boolean {
+    try {
```

Comment:
The `downloadFiles` function uses `curl` to download files, which may not be available on all systems. Consider using a more cross-platform solution, such as the `https` module in Node.js.[^1]
```suggestion
import * as https from 'https';
import * as fs from 'fs';

function downloadFiles(urls: string[], outputDir: string, outputFile: string): void {
    if (platform() !== 'linux') {
        return;
    }

    // Create output directory if it doesn't exist
    if (!fs.existsSync(outputDir)) {
        fs.mkdirSync(outputDir, { recursive: true });
    }

    urls.forEach(url => {
        const filePath = path.join(outputDir, outputFile || '');
        const file = fs.createWriteStream(filePath);

        https.get(url, (response) => {
            response.pipe(file);
            file.on('finish', () => {
                file.close();
            });
        }).on('error', (err) => {
            fs.unlink(filePath, () => {}); // Delete the file async
            console.error(`Error downloading ${url}: ${err.message}`);
        });
    });
}
```